gensim/models/wrappers/fasttext.py

```diff
@@ -363,13 +377,15 @@ def init_ngrams(self):
 
         ngram_weights = self.wv.syn0_all
 
-        for w, v in self.wv.vocab.items():
+        logger.info("loading vocabulary weights")
```

Comment:
Try to make the INFO messages more informative: loading how many weights, loading from where etc.

When there's a problem, it really helps when the log contains concrete numbers and messages. The problem often becomes apparent even without debugging.